docs/pages/ingesting/ingest_scripts/writing_scipts/writing_scripts.rst

```diff
@@ -0,0 +1,74 @@
+Writing Scripts
+==================
+
+When writing ingest scripts, there are two different ways to go about it: using existing ingest functions from `astrodb_utils` or using sqlalchemy commands. 
+
+
+Using Existing Ingest functions
+---------------------------------
+Using existing ingest functions helps streamline the process of writing an ingest script. 
+However, only few ingest functions exist, namely for sources, names, and instruments. 
+Therefore, if your data fits into one of these categories, it is recommended to use the existing functions. 
+
+Below is an example of how to use the `ingest_source` function to ingest source data into the database:
```

Comment:
link directly to the API documentation for the ingest_source function.
:py:function:`ingest_source`